docs/markdown.md

```diff
@@ -31,7 +31,245 @@ The front matter can also override the following [project configuration](./confi
+
+Cards can be used on their own, but they most often exist in a [grid](#grid). Cards can contain whatever you like, including text, images, charts, tables, inputs, and more.
+
+```html echo
+<div class="grid grid-cols-2">
+  <div class="card">
+    <h2>Lorem ipsum</h2>
+    <p>Id ornare arcu odio ut sem nulla pharetra. Aliquet lectus proin nibh nisl condimentum id venenatis a. Feugiat sed lectus vestibulum mattis ullamcorper velit. Aliquet nec ullamcorper sit amet. Sit amet tellus cras adipiscing. Condimentum id venenatis a condimentum vitae. Semper eget duis at tellus. Ut faucibus pulvinar elementum integer enim.</p>
+    <p>Et malesuada fames ac turpis. Integer vitae justo eget magna fermentum iaculis eu non diam. Aliquet risus feugiat in ante metus dictum at. Consectetur purus ut faucibus pulvinar.</p>
+  </div>
+  <div class="card" style="padding: 0;">
+    ${Inputs.table(industries)}
+  </div>
+</div>
+```
+
+<div class="tip">Remove the padding from a card if it contains only a table.</div>
+
+To place an input inside a card, first declare a detached input as a [top-level variable](./reactivity#top-level-variables) and use [`Generators.input`](./lib/generators#inputelement) to expose its reactive value:
+
+```js echo
+const industryInput = Inputs.select(industries.map((d) => d.industry), {unique: true, sort: true, label: "Industry:"});
+const industry = Generators.input(industryInput);
+```
+
+Then, insert the input into the card:
+
+```html echo
+<div class="card" style="display: flex; flex-direction: column; gap: 1rem;">
+  ${industryInput}
+  ${resize((width) => Plot.plot({
+    width,
+    y: {grid: true, label: "Unemployed (thousands)"},
+    marks: [
+      Plot.areaY(industries.filter((d) => d.industry === industry), {x: "date", y: "unemployed", fill: "var(--theme-foreground-muted)", curve: "step"}),
+      Plot.lineY(industries.filter((d) => d.industry === industry), {x: "date", y: "unemployed", curve: "step"}),
+      Plot.ruleY([0])
+    ]
+  }))}
+</div>
+```
+
+## Notes
+
+The `note`, `tip`, `warning`, and `caution` classes can be used to insert labeled notes (also known as callouts) into prose. These are intended to emphasize important information that could otherwise be overlooked.
+
+<div class="note">This is a note.</div>
+
+```html run=false
+<div class="note">This is a note.</div>
+```
+
+<div class="tip">This is a tip.</div>
+
+```html run=false
+<div class="tip">This is a tip.</div>
+```
+
+<div class="warning">This is a warning.</div>
+
+```html run=false
+<div class="warning">This is a warning.</div>
+```
+
+<div class="caution">This is a caution.</div>
+
+```html run=false
+<div class="caution">This is a caution.</div>
+```
+
+Markdown is not supported within HTML, so if you want rich formatting or links within a note, you must write it as HTML. (In the future, we may add support for notes within Markdown.)
```

Comment:
I wrote this poorly, but what I meant was that in Markdown, within an HTML block (`<div>…</div>`), the contents of the HTML block are interpreted as HTML rather than Markdown. So if you say this

```md
<div>This is *HTML*.</div>
```

the asterisks are interpreted literally (as in HTML), and you see this on the resulting page:

> This is \*HTML\*.

If you want the contents of an HTML block to be interpreted as Markdown (as italics) instead of HTML, you need to use blank lines

```md
<div>

This is *Markdown*.

</div>
```

which renders as

> This is *Markdown*.

This is per the [CommonMark specification](https://spec.commonmark.org/0.30/#html-blocks).

The parenthetical comment on “support for notes within Markdown” was me thinking about adding Markdown-specific syntax for notes, as [GitHub does](https://github.com/orgs/community/discussions/16925) and as [VitePress does](https://vitepress.dev/guide/markdown#custom-containers).

I’ll try to rewrite this…